spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'ingredients index' do
+  it "lists all ingredients and their cost" do
+    ingredient_1 = Ingredient.create(name: "Ground Beef", cost: 2)
+    ingredient_2 = Ingredient.create(name: "Salt", cost: 4)
+
+    visit "/ingredients"
+
+    expect(page).to have_content(ingredient_1.name)
```

Comment:
Nice job testing. Using `within` blocks can make for more robust tests. Also you will want to test things as you have them appear in your view: example
```ruby
within "#ingredients" do 
  expect(page).to have_content("#{ingredient_1.name}: #{ingredient_1.cost}")
end
```